packages/core/src/layout/ItemCard/ItemCard.tsx

```diff
@@ -61,8 +61,8 @@ export const ItemCard: FC<ItemCardProps> = ({
         <Typography variant="h6">{title}</Typography>
       </div>
       <div className={classes.content}>
-        {tags?.map(tag => (
-          <Chip label={tag} key={tag} />
+        {tags?.map((tag, i) => (
+          <Chip label={tag} key={`tag-${i}`} />
```

Comment:
Coupled it with the index, to handle the duplicate key warning when array of tags is having two are more tags with the same value.